udata_gouvfr/harvesters/maaf.py

```diff
@@ -54,10 +60,14 @@
 XSD_PATH = os.path.join(os.path.dirname(__file__), 'maaf.xsd')
 
 SSL_COMMENT = '''
-Le site exposant les données est protégé par un certificat délivré par l'IGC/A (IGC officielle de l'État).
-Une exception de sécurité peut apparaître si votre navigateur ne reconnait pas cette autorité :
-vous trouverez  la procédure à suivre pour éviter une telle alerte à l'adresse :
-http://www.ssi.gouv.fr/fr/anssi/services-securises/igc-a/modalites-de-verification-du-certificat-de-l-igc-a-rsa-4096.html
+Le site exposant les données est protégé par un certificat délivré par
+l'IGC/A (IGC officielle de l'État).
+Une exception de sécurité peut apparaître si votre navigateur ne reconnait
+pas cette autorité :
+vous trouverez  la procédure à suivre pour éviter une telle alerte
+à l'adresse :
+http://www.ssi.gouv.fr/fr/anssi/services-securises/igc-a/
+modalites-de-verification-du-certificat-de-l-igc-a-rsa-4096.html
```

Comment:
URL Shouldn't be split as it should be parsed (will be displayed in the resource popup)